docs/core/tools/dotnet-workload-sets.md

```diff
@@ -0,0 +1,165 @@
+* Check your current update mode and workload-set version.
+
+And you can still choose to install, update, or restore to the latest version of each individual workload, ignoring workload sets.
+
+## Prerequisites
+
+* [.NET 8.0.400 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/8.0) or later.
+
+  In 8.0.400 SDK, the `dotnet workload` commands are in workload-set update mode only when it's explicitly selected. Starting in [.NET 9 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/9.0), `dotnet workload` commands are in workload-set update mode by default.
+
+## "Pin" the install command
+
+A `dotnet workload install` command with the `--version` option "pins" the `install` command in workload set update mode with the specified workload set version.
+<!--todo: install without --version in workload set update mode pins to latest workload set?-->
+The command no longer automatically installs the newest workload based on loose manifests. For example:
+
+1. ```dotnetcli
+   dotnet workload install aspire --version 9.0.100-preview.7.24414.1
+   ```
+
+   When this command runs:
+
+   * It selects workload set update mode if that isn't already selected.
+   * It gets the workload set that has the specified version.
+   * From the workload set, it gets the manifest version of the specified workload.
+   * It installs the manifest version of the workload.
+   * It stays in workload-set update mode when it's done.
+
+1. ```dotnetcli
+   dotnet workload install maui-ios
+   ```
+
+   This command installs the workload set version `9.0.100-preview.7.24414.1`, since the preceding `install` command example pinned that workload set.
+
+Using `--version` with either `install` or `update` pins `install` to the specified version, but `update` is only pinned to workload-set update mode, not to a workload-set version. If you then run `dotnet workload update` without the `--version` option, the `update` command:
+
+* Updates workloads to the latest available workload-set version.
+* "Unpins" the `install` command.
+* Stays in workload-set update mode.
+
+## Update using latest workload set
+
+To update installed workloads to the latest workload-set version available on the configured feeds, run the following commands:
+
+1. ```dotnetcli
+   dotnet workload config --update-mode workload-set
+   ```
+
+   The preceding command is necessary only if you are currently in manifests update mode. If you don't know, [check the current update mode](#check-the-update-mode-and-version). Workload-set update mode is the default starting in .NET 9 SDK.
+
+1. ```dotnetcli
+   dotnet workload update
+   ```
+
+   In workload set update mode, this command updates workloads to the latest workload set version, unless you have specified the workload set version in global.json.
+
+## Update to a workload-set version
+
+To specify a workload set version to update to when you're not specifying it in global.json, use the `--version` option of the `update` command. For example:
```

Comment:
Where should we inform customers that the workload sets will typically align with the .Net sdk releases so if they are using an 8.0.4xx SDK, they'll want an 8.0.4xx workload set. BTW, there's a new feature coming that will let you search for workload set versions that are available.

Reply:
I think there should be a 'choosing a workload set version' subsection in this doc where we could say this. Main idea would be 

* each SDK has a matching workload set version
* in the future we'll give you ways to list workload set versions to see what's available and what they contain

And then each command that has a `--version` could link to that section heading.